README.md

```diff
@@ -21,6 +25,7 @@ See the [CHANGELOG](CHANGELOG.md).
 
 - [Contributing](docs/CONTRIBUTING.md)
 - [Technical Documentation](docs/DOCUMENTATION.md)
+- [Report an issue](https://issues.jenkins-ci.org/browse/JENKINS-53417)
```

Comment:
Wrong link

Reply:
@oleg-nenashev I planned to direct people to see this long term plans epic where we had issues created and they can add more